cartoframes/core/cartodataframe.py

```diff
@@ -24,16 +46,45 @@ def from_features(cls, features, **kwargs):
         return cls(gdf)
 
     def to_carto(self, *args, **kwargs):
+        """
+        Upload a CartoDataFrame to CARTO.
+        Equivalent to :py:meth:`to_carto <cartoframes.io.to_carto>`.
+
+        Examples:
+
+            .. code::
+                cdf.to_carto(if_exists='replace')
```

Comment:
Let's include in the code examples some more context, something like this could work:

```python
from cartoframes import CartoDataFrame

cdf = CartoDataFrame.from_file('nybb.shp')
cdf.to_carto(if_exists='replace')
```